frontend/src/components/ImagesComponent.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FunctionComponent, ReactElement, useState } from "react";
+import styled from "styled-components";
+import { SEA_FOAM } from "../constants/colors";
+
+
+interface ImagesProp {
+  images: string[],
+}
+
+interface ImageProp {
+  highlight: boolean,
+}
+
+const Image = styled.img`
+  border-radius: 20px;
+  width: 100%;
+  height: 100%;
+  transform: scale(0.8);
+
+  :hover {
+    cursor: pointer;
+    opacity: 0.8;
+  }
+
+  ${({highlight}: ImageProp) => highlight && `
+    border: 10px solid ${SEA_FOAM};
+  `};
+`;
+
+const ImagesComponent: FunctionComponent<ImagesProp> = ({ images })  : ReactElement => {
```

Comment:
Can we rename this to something more specific to what it does? ImagesComponent doesn't really mean anything

Reply:
Sorry just getting around to this, umm "ImageGrid"?